server/server.go

```diff
@@ -544,6 +546,17 @@ func (s *Server) handlePublishWithoutResponse(r *http.Request, v *visitor) (*mes
 	if err != nil {
 		return nil, err
 	}
+
+	var v_billing *visitor
+	if strings.HasPrefix(t.ID, unifiedpushTopicPrefix) {
+		v_billing := t.getBillee()
+		if v_billing != nil {
+			// instant reject and won't even store it if there's no one registered for a UP topic in the past some time
+	// need to find error code for device not available try again later
+			return nil, errHTTPInternalError
+		}
+	}
```

Comment:
If there is no last subscriber, or there was a subscriber very long ago, what should ntfy do? 
1. Cache it anyway but bill it to the app server - inconsistent
2. Reject it. Which status code? 500?

Reply:
*should be == not !=

Reply:
I think this is kind of up to the UP spec, no? "What should the push server do if there is no one listening" seems like a spec question.

Obviously the easiest thing would be to reject it. Everything else would cause lots of headscratching.

The response code may be https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412?

Reply:
Good point